### DISCUSSION TITLE
Questions about watch history capabilities

### DISCUSSION THREAD

**Discussion by vynxc** [crunchy-labs/crunchyroll-rs#37] [Q&A]
I didn't see any implementation to set watch history, if this doesn't exist already is it planned? Just come context, I'm looking to create a cli tool to copy watch history from one account/profile to another.

**Comment by bytedream** [comment]
Setting a episode as watched is a bit ugly. The only way to do it is via the [set_playhead](https://docs.rs/crunchyroll-rs/latest/crunchyroll_rs/media/struct.Episode.html#method.set_playhead) method.
I think something like this should work:
```rust
episode.set_playhead(episode.duration.num_seconds() as u32).await
```
